src/AddTodoForm.js

```diff
@@ -1,11 +1,28 @@
 import React from "react";
 
-function AddTodoForm() {
+function AddTodoForm(props) {
+  const handleAddTodo = (event) => {
+    event.preventDefault();
+    console.log(event);
+
+    const todoTitle = event.target.elements.title.value;
+    console.log(todoTitle);
```

Comment:
You can directly access the input value with `event.target.title.value` 
You can also remove the console.log statements.